src/utils/steps.ts

```diff
@@ -0,0 +1,26 @@
+import { StepType } from "../steps/UploadFlow"
```

Comment:
moving useRsiInitialStep logic here is fine by me but might make it more difficult to get this merged if we're going to open a PR to the official repo